src/Models/EventModel.php

```diff
@@ -169,7 +169,7 @@ public function searchPastEvents(?string $searchTerm): array
      * @return array An array of Events based on category ID and previous X months from current date.
      */
 
-    public function getEventsByCategoryId(string $categoryId, int $previousMonths = 0): array
+    public function getEventsByCategoryId(string $categoryId, int $previousMonths = 9999): array
```

Comment:
I don't like this. Is there a better way?

Reply:
Would suggest leaving the original default, but pass in the number of previous months you want when this method gets called in the controller.

Maybe don't go for 9999, maybe do like 5 years worth?

Reply:
done